### DISCUSSION TITLE
Current progress

### DISCUSSION THREAD

**Discussion by lelouvincx** [TinhHoaSolutions-EzHR/chatbot#3] [Show and tell]

**Comment by lelouvincx** [comment]
I am currently working on writing SRS and drawing app archietcture with @tuantran0910 boi` q

**Reply by greyyT** [reply]
Is that necessary? Since SRS is considered fat.

**Reply by lelouvincx** [reply]
Yes, because Mr. May Noi needs it.

**Comment by tuantran0910** [comment]
I think you fat

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: lelouvincx ***@***.***>
Sent: Wednesday, October 30, 2024 9:27:59 PM
To: TinhHoaSolutions-EzHR/chatbot ***@***.***>
Cc: TRẦN NGỌC TUẤN ***@***.***>; Mention ***@***.***>
Subject: [TinhHoaSolutions-EzHR] Drawing app's architecture (Discussion #3)


With @tuantran0910<https://github.com/tuantran0910> boi`

Issue #2<https://github.com/TinhHoaSolutions-EzHR/chatbot/issues/2>

**Reply by lelouvincx** [reply]
I think u fatter
